apache-activemq-mixin/mixin.libsonnet

```diff
@@ -0,0 +1,4 @@
+(import 'dashboards/dashboards.libsonnet') +
+(import 'alerts/alerts.libsonnet') +
+(import 'config.libsonnet') +
+(import 'dashboards/apache-activemq-logs-overview.libsonnet')
```

Comment:
any reason why this is called out special?

Reply:
Not particularly, since the logs dashboard is autogenerated i pulled it out when developing, but i can add this bit back with all the others

Reply:
good to resolve